instructor/migrations/0002_auto_20151002_1248.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
```

Comment:
It would be better if you rename this migration file to something more helpful according to what the migration does.
Something like `0002_new_models.py` or even more specific